frontend/express/public/core/token-manager/javascripts/countly.views.js

```diff
@@ -0,0 +1,229 @@
+                        newEndpoints.push(obj);
+                    }
+                });
+                endpoints = JSON.stringify(newEndpoints);
+
+                if (self.tokenUsage === "1") {
+                    if (doc.selectApps.length > 0) {
+                        selectApps = doc.selectApps.join(",");
+                    }
+                }
+                if (self.tokenExpiration === "1") {
+                    if (doc.selectTime === "hours") {
+                        ttl = doc.timeInput * 3600;
+                    }
+                    else if (doc.selectTime === "days") {
+                        ttl = doc.timeInput * 3600 * 24;
+                    }
+                    else if (doc.selectTime === "months") {
+                        ttl = doc.timeInput * 3600 * 24 * 30;
+                    }
+                }
+                countlyTokenManager.createTokenWithQuery(doc.description, endpoints, doc.checkboxMultipleTimes, selectApps, ttl, function() {
+                    self.$emit("create");
+                });
+            }
+        },
+        props: {
+            controls: {
+                type: Object
+            }
+        },
+    });
+
+    var TokenManager = countlyVue.views.create({
+        template: CV.T('/core/token-manager/templates/token-manager.html'),
+        components: {
+            "main-drawer": TokenDrawer
+        },
+        mixins: [countlyVue.mixins.hasDrawers("main")],
+        data: function() {
+            return {
+                tableData: []
+            };
+        },
+        mounted: function(row) {
+            var self = this;
+            $.when(countlyTokenManager.initialize()).then(function() {
+                self.prepareTableData(row);
+            });
+        },
+        methods: {
+            refresh: function(row) {
+                var self = this;
+                $.when(countlyTokenManager.initialize()).then(function() {
+                    self.prepareTableData(row);
+                });
+            },
+            prepareTableData: function(row) {
+                var tableData = countlyTokenManager.getData();
+                for (var j = 0; j < tableData.length; j++) {
+                    if (tableData[j]._id === countlyGlobal.auth_token) {
+                        tableData.splice(j, 1);
+                    }
+                }
+                for (var i = 0; i < tableData.length; i++) {
+                    row = tableData[i];
+                    if (row.ttl && ((row.ends * 1000) - Date.now()) < 0) {
+                        row.status = "expired";
+                    }
+                    else {
+                        row.status = "active";
+                    }
+                    if (row.ttl) {
+                        row.ttlDate = countlyCommon.getDate(row.ends);
+                        row.ttlTime = countlyCommon.getTime(row.ends);
+                    }
+                    else {
+                        row.ttlDate = "No expiry";
```

Comment:
Could you please replace this with localization "token_manager.table.not-expire" 🙂?